tests/gemv.cpp

```diff
@@ -0,0 +1,164 @@
+#include <experimental/linalg>
+#include <experimental/mdspan>
+
+#include <vector>
+#include "gtest/gtest.h"
+#include <iostream>
+
+namespace {
+  using std::experimental::basic_mdspan;
+  using std::experimental::dynamic_extent;
+  using std::experimental::extents;
+  using std::experimental::layout_left;
+  using std::experimental::matrix_product;
+  using std::experimental::transpose_view;
+  using std::cout;
+  using std::endl;
+
+  template<class MdspanType, class Scalar>
+  struct FillMatrix {
+    static void fill(MdspanType A, const Scalar startVal);
+  };
+
+  template<class MdspanType>
+  struct FillMatrix<MdspanType, int> {
```

Comment:
This is perfectly fine, but I realize that a more modern way of doing this might be just by overloading, instead of this struct thing:
```cpp
template<class Layout, class Extents, class Accessor>
void fill(basic_mdspan<double, Layout, Extents, Accessor> A, const int startVal)
{ /* ... */ }
```